.forge-snapshots/donate gas with 1 token.snap

```diff
@@ -1 +1 @@
-95961
\ No newline at end of file
+97672
```

Comment:
these look large, but our gas tests arent actually just checking the call to the contract... theyre including all the logic in the `PoolDonateTest` or `PoolSwapTest` contracts.